### PR TITLE
Normalize \r style multiline properties correctly

### LINKED ISSUE #289: Some Expressions in the premade calendar Screen aren't recognized as multiline properties

### PR DESCRIPTION
**Pull request by lesaltzm**
fixes #289

### CODE REVIEW

**Review by suskumar-MSFT — APPROVED** [review]
:shipit:

File: src/PAModel/PAConvert/Yaml/YamlWriter.cs [hunk 1]
```diff
@@ -149,5 +145,10 @@ private void WriteIndent()
                 _text.Write(Indent);
             }
         }
+
+        public static string NormalizeNewlines(string x)
```
Comment (MikeStall) [inline src/PAModel/PAConvert/Yaml/YamlWriter.cs:149]: >NormalizeNewlines [](http://example.com/codeflow?start=29&length=17)

Can we add some more dedicated tests for this?

There are so many corner cases here. 

\r, \n, \r\n --> single
\r\r,  \n\n --> double
